addons/account/tests/test_account_move_reconcile.py

```diff
@@ -2692,7 +2742,7 @@ def test_reconcile_cash_basis_workflow_multi_currency(self):
 
         self.assertAmountsGroupByAccount([
             # Account                               Balance     Amount Currency
-            (self.cash_basis_transfer_account,      5.57,       -0.002),
+            (self.cash_basis_transfer_account,      0.0,        -0.002),
```

Comment:
The lines created since the last `assertAmountsGroupByAccount` are
* Company: `5.56 + 0.0 + 5.55 + 0.0 + 0.0 + 0.0 = 11.11`
* Foreign: `11.109 + 0.003 + 11.107 + 0.003 + 0.002 + 0.0 = 22.224`

The amount in foreign currency matches the sum compared to the previous amounts, but not in company currency